adal/src/main/java/com/microsoft/aad/adal/TokenCacheItem.java

```diff
@@ -82,6 +84,9 @@ public TokenCacheItem() {}
         this.mIsMultiResourceRefreshToken = tokenCacheItem.getIsMultiResourceRefreshToken();
         this.mTenantId = tokenCacheItem.getTenantId();
         this.mFamilyClientId = tokenCacheItem.getFamilyClientId();
+        if (tokenCacheItem.isExtendedLifetimeValid()) {
```

Comment:
by that justification we should not copy the expiresOn either.

---
In reply to: [71611019](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/676#discussion_r71611019) [](ancestors = 71611019)